swap_meet/vendor.py

```diff
@@ -1,2 +1,57 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
```

Comment:
Since the prior `if` block had a return the `else` is not needed

```suggestion
        self.inventory.remove(item)
        return item
```

Reply:
Thanks for the reminder!